client/plots/disco/label/Labels.ts

```diff
@@ -9,10 +9,15 @@ export default class Labels extends Ring<Label> {
 	elementsToDisplay: Array<Label> = []
 
 	private hasPrioritizedGenes: boolean
-	private filteredPrioritizedGenesList: Array<Label> = []
 	private overlapAngle: number
+	private prioritizeGeneLabelsByGeneSets: boolean
 
-	constructor(settings: Settings, elements: Array<Label>, hasPrioritizedGenes: boolean) {
+	constructor(
+		settings: Settings,
+		elements: Array<Label>,
+		hasPrioritizedGenes: boolean,
+		prioritizeGeneLabelsByGeneSets: boolean
```

Comment:
@aacic there is also a duplicate argument here that could have used `settings.prioritizeGeneLabelsByGeneSets`?

Reply:
Yes. I've removed the duplicate.